### ISSUE TITLE
sysinfo error for ios

### ISSUE THREAD

**Issue by goozang-records** [sunshine-protocol/sunshine-identity-ui#1]
[error log](https://gist.github.com/4meta5/6ea719a4a9a588a5b8205b8d9b1d9fac)

**Comment by goozang-records** [comment]
The problem is that https://github.com/guillaumegomez/sysinfo does not support ios

**Comment by shekohex** [comment]
it seems that [`sc-service`](https://github.com/paritytech/substrate/blob/master/client/service) uses this crate `sysinfo` but it ignores it when it comes to [`android`](https://github.com/paritytech/substrate/blob/master/client/service/src/metrics.rs#L189) I guess a fix is that it should ignore iOS too.
also, sysinfo should be added to cargo.toml to `sc-serivce` like this
```toml
[target.'cfg(all(any(unix, windows), not(all(target_os = "android", target_os = "ios"))))'.dependencies]
sysinfo = "0.13.3"

```

**Comment by goozang-records** [comment]
closed in #4
